.github/workflows/test-packages-action-macos.yml

```diff
@@ -70,7 +70,10 @@ jobs:
 
   generate-matrix:
     name: Generate Matrix
-    runs-on: ${{ github.event.repository.private && fromJSON('["self-hosted", "linux", "x86_64"]') || 'ubuntu-latest' }}
+    runs-on:
+      - self-hosted
+      - linux
+      - x86_64
```

Comment:
Why are we changing this part? I think we talked about it but remind me please...

Reply:
it needs to be able to access the s3 bucket, so it needs our runners for the creds